contrib/mod_rewrite.c

```diff
@@ -539,12 +564,11 @@ static unsigned char rewrite_match_cond(cmd_rec *cmd, config_rec *cond) {
       rewrite_log("rewrite_match_cond(): checked lexical LT cond: %s > %s: %d",
         cond_str, (char *) cond->argv[1], res);
 
-      if (!negated) {
-        return (res < 0 ? TRUE : FALSE);
-
-      } else {
+      if (negated == TRUE) {
```

Comment:
The human brain has a much easier time processing positives _vs_ negatives, thus I rewrote these conditionals to use the positive case first, and the negative case as the `else` branch.